lib/features/mailbox/presentation/mailbox_controller.dart

```diff
@@ -533,29 +536,35 @@ class MailboxController extends BaseMailboxController {
     }
   }
 
-  void pressMailboxSelectionAction(BuildContext context, MailboxActions actions,
-      List<PresentationMailbox> selectionMailbox) {
+  void pressMailboxSelectionAction(
+      BuildContext context, 
+      MailboxActions actions,
+      List<PresentationMailbox> selectedMailboxList
+  ) {
     switch(actions) {
       case MailboxActions.delete:
-        _openConfirmationDialogDeleteMailboxAction(context, selectionMailbox.first);
```

Comment:
:)

Reply:
dm

Reply:
Do not laugh. I used another function to handle it.